django/contrib/postgres/fields/jsonb.py

```diff
@@ -1,188 +1,10 @@
-import json
-
-from psycopg2.extras import Json
-
-from django.contrib.postgres import forms, lookups
-from django.core import exceptions
-from django.db.models import (
-    Field, TextField, Transform, lookups as builtin_lookups,
-)
-from django.utils.translation import gettext_lazy as _
-
-from .mixins import CheckFieldDefaultMixin
+from django.db.models import JSONField as BuiltinJSONField
 
 __all__ = ['JSONField']
 
 
-class JsonAdapter(Json):
-    """
-    Customized psycopg2.extras.Json to allow for a custom encoder.
-    """
-    def __init__(self, adapted, dumps=None, encoder=None):
-        self.encoder = encoder
-        super().__init__(adapted, dumps=dumps)
-
-    def dumps(self, obj):
-        options = {'cls': self.encoder} if self.encoder else {}
-        return json.dumps(obj, **options)
-
-
-class JSONField(CheckFieldDefaultMixin, Field):
-    empty_strings_allowed = False
-    description = _('A JSON object')
-    default_error_messages = {
-        'invalid': _("Value must be valid JSON."),
-    }
-    _default_hint = ('dict', '{}')
-
-    def __init__(self, verbose_name=None, name=None, encoder=None, **kwargs):
-        if encoder and not callable(encoder):
-            raise ValueError("The encoder parameter must be a callable object.")
-        self.encoder = encoder
-        super().__init__(verbose_name, name, **kwargs)
-
-    def db_type(self, connection):
-        return 'jsonb'
-
+class JSONField(BuiltinJSONField):
```

Comment:
Maybe @felixxm  or @carltongibson can guide, but I believe it'd be good practice to use a `warnings.warn` in `__init__`, although a deprecation timeline has not been determined for `django.contrib.postgres.field.JSONField`.